lib/ansible/plugins/inventory/__init__.py

```diff
@@ -150,6 +150,12 @@ class BaseInventoryPlugin(AnsiblePlugin):
     """ Parses an Inventory Source"""
 
     TYPE = 'generator'
+
+    # This is not actually private.
+    # 3rd party plugins redefine this to
+    # use custom group name sanitization
+    # since constructed features enforce
+    # it by default.
```

Comment:
it is 'private' as it should not be called from outside the plugin, but each plugin can define what the method should be implemented as.

Reply:
Fixed that